samples/General/enforce-tag-and-value/README.md

```diff
@@ -0,0 +1,22 @@
+## Deny for tags and their values
+Enforces a required tag and its value. Does not apply to resource groups.
+
+
+
+## Try with PowerShell
+````powershell
+$rg = Get-AzureRmResourceGroup -Name 'PriyaTestGroup'
+
+$definition = Get-AzureRmPolicyDefinition | Where-Object { $_.Properties.DisplayName -eq 'Enforce tag and its value' }

```

Comment:
This example works because this policy definition has already been added as a built-in policy, so it already exists in the list of policies returned by Get-AzureRmPolicyDefinition. However, the intent of this samples repository is to show how to create _new_ policy definitions. So, even though the definition already exists, the example should show how to create it from scratch. (This is admittedly a slightly strange thing to do - no one would want to exactly duplicate a built-in policy - but it gives customers a starting point if they want to slightly modify one of the built-in policies). 

So, this should be New-AzureRmPolicyDefinition, similar to your cli example. Something like this:
`
$definition = New-AzureRmPolicyDefinition -Name "enforce-tag-and-value" -DisplayName "Deny for tag and value" -description "Enforces a required tag and its value. Does not apply to resource groups." -Policy 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/General/enforce-tag-and-value/azurepolicy.rules.json' -Parameter 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/General/enforce-tag-and-value/azurepolicy.parameters.json'`